spec.html

```diff
@@ -35141,6 +35141,9 @@ <h1>
           <emu-note>
             <p>This algorithm differs from <a href="https://unicode.org/reports/tr44/#Matching_Symbolic">the matching rules for symbolic values listed in UAX44</a>: case, <emu-xref href="#sec-white-space">white space</emu-xref>, U+002D (HYPHEN-MINUS), and U+005F (LOW LINE) are not ignored, and the `Is` prefix is not supported.</p>
           </emu-note>
+          <emu-note>
+            <p>The spellings of entries in these tables (including casing) were chosen to match the first occurrence of each property in the files PropertyAliases.txt and PropertyValueAliases.txt in the Unicode Character Database at the time each entry was added to this specification. However, because the precise spellings in those files are not guaranteed to be stable, implementations are required to follow this table rather than those files.</p>
```

Comment:
LGTM 👍 Some nits/thoughts:

- Let’s add links to these files? https://unicode.org/Public/UCD/latest/ucd/PropertyAliases.txt and https://unicode.org/Public/UCD/latest/ucd/PropertyValueAliases.txt are stable URLs.
- Should the file names be wrapped in `<code>`?

Reply:
So like this?

```suggestion
            <p>The spellings of entries in these tables (including casing) were chosen to match the first occurrence of each property in the files <a href="https://unicode.org/Public/UCD/latest/ucd/PropertyAliases.txt"><code>PropertyAliases.txt</code></a> and <a href="https://unicode.org/Public/UCD/latest/ucd/PropertyValueAliases.txt"><code>PropertyValueAliases.txt</code></a> in the Unicode Character Database at the time each entry was added to this specification. However, because the precise spellings in those files are not guaranteed to be stable, implementations are required to follow this table rather than those files.</p>
```

Reply:
:shipit:

Reply:
Would that also apply to the other UCD file references (cf. [#2594](https://github.com/tc39/ecma262/pull/2594/files))?

Reply:
@gibson042 Yes, please do.

Reply:
Done.